packages/flutter/test/widgets/table_test.dart

```diff
@@ -297,6 +297,31 @@ void main() {
     expect(boxG1, isNot(equals(boxG2)));
   });
 
+  // Regression test for https://github.com/flutter/flutter/issues/27083
+  testWidgets('Really small deficit double precision error', (WidgetTester tester) async {
+    const SizedBox cell = SizedBox(width: 16, height: 16);
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: Table(
+          children: const <TableRow>[
+            TableRow(
+              children: <Widget>[
+                cell, cell, cell, cell, cell, cell,
+              ],
+            ),
+            TableRow(
+              children: <Widget>[
+                cell, cell, cell, cell, cell, cell,
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+    expect(true, true);
```

Comment:
This looks very odd. Is that necessary? If so, probably deserves a comment.

Reply:
Not really necessary, I can leave it out. The test will run forever without the fix applied so I felt it was necessary to put something in there...

Reply:
Maybe just a comment that the failing condition is never terminates?

Reply:
Done